templates/base.html

```diff
@@ -53,7 +53,7 @@
                         <div class="hide-for-small-only top-bar nav">
                             <div class="top-bar-left">
                                 <ul class="dropdown menu nav" data-dropdown-menu>
-                                    <li><a class="active" href="{% url 'home' %}">HOME</a></li>
```

Comment:
I think `active` is a class from Foundation that is applied to whichever link is the one that's currently selected. Are you want to remove this?